packages/core/card.ts

```diff
@@ -330,25 +336,29 @@ export class Card {
 
   @Memoize()
   async field(name: string): Promise<FieldCard> {
-    if (this.csFields) {
-      if (name in this.csFields) {
-        return await this.container.instantiate(
-          FieldCard,
-          merge({ data: { type: 'cards' } }, { data: this.csFields[name] }),
-          name,
-          this,
-          this.reader,
-          this.modules,
-          this.container
-        );
-      }
-    }
-    let parent = await this.adoptsFrom();
-    if (parent) {
-      return await parent.field(name);
+    let data;
+
+    if (name === 'csAdoptsFrom') {
```

Comment:
If we're going to let people treat `csAdoptsFrom` the same as the user defined fields (and I think that's a good idea), it needs to have a FieldCard defining it.